### PR TITLE
Set logging level from the command line (and propagate level using multiprocessing's logger)

### PR DESCRIPTION
**Pull request by jordis-ai2**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -130,6 +130,17 @@ def get_args():
...
 
+    parser.add_argument(
+        "-l",
+        "--log_level",
+        default="info",
+        type=str,
+        required=False,
+        help="sets the log_level. it must be one of {{{}}}.".format(
+            ", ".join(HUMAN_LOG_LEVELS)
+        ),
```
Comment (Lucaweihs) [inline main.py:141]: Why have `{{{}}}` instead of just `{}`?

**Review by Lucaweihs — COMMENTED** [review]
Looks reasonable to me but I must admit I'm not very well versed in logging code. If you've tested it in multiple settings and it works then I'm happy.

File: main.py (hunk 1 above)
Comment (jordis-ai2) [reply main.py:141]: I just thought it would be nice to show the possible options as a set, but I'll just list them.

**Comment by jordis-ai2** [comment]
Messages can still get mixed-up (https://docs.python.org/3.6/library/multiprocessing.html#logging), but it works well in most cases... It's the least invasive change I found to provide the functionality (I"m learning about logging as I write this code).

**Pull request merged**